e2e-test/wds_azure_e2etest.py

```diff
@@ -13,8 +13,9 @@
 wds_upload=True

```

Comment:
I renamed this file since there are now more tests in the folder and this helps make sure no one gets confused what e2e test this is. open to feedback on other possible naming!

Reply:
oh nice, that's getting a head start on https://broadworkbench.atlassian.net/browse/AJ-1398!